eclass/libtool.eclass

```diff
@@ -25,6 +25,70 @@ esac
 
 inherit toolchain-funcs
 
+# @ECLASS_VARIABLE: LIBTOOL
+# @DEFAULT_UNSET
+# @DESCRIPTION:
+# The default libtool implementation. If unset GNU libtool will be used in most
+# cases.
+
+# @FUNCTION: eslibtool
+# @USAGE: [flag]
+# @DESCRIPTION:
+# If the LIBTOOL variable in the user's environment is set and contains one of
+# the slibtool symlinks it will be modified as required by the ebuild.
+#
+# If the MAKE or MAKEFLAGS environment variables are set they will also be
+# modified accordingly.
+#
+# The flag should '-shared', '-static' or unset to use the equivelent slibtool
+# symlink. With no arguments both shared and static libraries will be enabled.
+eslibtool() {
+	if [[ -n "${LIBTOOL}" ]]; then
```

Comment:
How about fast exit to avoid the extra level of indentation, that exists across the whole function, here?
```suggestion
	if [[ -z "${LIBTOOL}" ]]; then return; fi
```